src/components/dialogs/network-modifications/line-attach-to-voltage-level/line-attach-to-voltage-level-form.js

```diff
@@ -192,7 +179,9 @@ const LineAttachToVoltageLevelForm = ({
                     currentNode={currentNode}
                     studyUuid={studyUuid}
                     onCreateVoltageLevel={onVoltageLevelCreationDo}
-                    editData={voltageLevelToEdit}
+                    editData={
```

Comment:
type of 'undefined' is not an object
but in VoltageLevelCreationDialog we have editData: PropTypes.object,